account_invoice_import/views/account_invoice_import_config.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2015-2017 Akretion (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+  <record id="account_invoice_import_config_form" model="ir.ui.view">
+    <field name="name">account.invoice.import.config.form</field>
+    <field name="model">account.invoice.import.config</field>
+    <field name="arch" type="xml">
+      <form string="Invoice Import Configuration">
+        <div class="oe_button_box" name="button_box">
+          <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
+            <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
+          </button>
+        </div>
+        <group name="main">
+          <field name="name"/>
+          <field name="company_id" groups="base.group_multi_company"/>
+        </group>
+        <group string="Accounting Parameters" name="accounting">
+          <field name="invoice_line_method"/>
+          <field name="account_id" attrs="{'invisible': [('invoice_line_method', 'not in', ('1line_no_product', 'nline_no_product'))], 'required': [('invoice_line_method', 'in', ('1line_no_product', 'nline_no_product'))]}"/>
```

Comment:
IMHO this should be visible always so users can set their preferred account

Reply:
if product, account herit from product.